config/cable.yml

```diff
@@ -1,5 +1,6 @@
 development:
-  adapter: async
+  adapter: redis
+  url: redis://localhost:6379/1
```

Comment:
Have we validated that we are going to be using ActionCable, and if so that we are going to be using Redis as an adapter?

Reply:
We haven't validated it -- but to be honest I don't know the alternatives. Let's chat about this?

Reply:
Are we able to come back to this decision at a later time? I'am trying to figure out what is blocking and what isn't. I'll certainly want to create an issue/discussion to talk about this more thoroughly though